home-assistant-voice.yaml

```diff
@@ -1428,26 +1404,58 @@ speaker:
     channel: stereo
     timeout: never
     buffer_duration: 100ms
+    audio_dac: aic3204_dac
 
-media_player:
-  - platform: nabu
```

Comment:
Should this code be deleted from https://github.com/esphome/home-assistant-voice-pe/tree/dev/esphome/components if we're no longer using it?